### PR TITLE
Add more explicit information about LDAP configuration

### LINKED ISSUE #181: Add more information on the LDAP group configure
There is a bug in the LDAP group search which doesn't consider the LDAP group Filter item if it is not well configured. It is fixed after Harbor 2.1.3, but it might looks like a breaking change,  some configuration works before 2.1.3, but doesn't work after 2.1.3.

Could you please add information to the LDAP configure item?

https://goharbor.io/docs/2.0.0/administration/configure-authentication/ldap-auth/

----
```
LDAP Group Base DN: The base DN from which to lookup a group in LDAP/AD. For example, ou=groups,dc=example,dc=com.
```
Change to
```
LDAP Group Base DN: The base DN from which to lookup a group in LDAP/AD. For example, ou=groups,dc=example,dc=com. This field cannot be empty.
```

----
```
LDAP Group Filter: The filter to search for LDAP/AD groups. For example, objectclass=groupOfNames.
```
Change to :
```
LDAP Group Filter: The filter to search for LDAP/AD groups. for OpenLDAP: objectclass=groupOfNames. for Active Directory: objectclass=group. This field cannot be empty.
```

----
```
LDAP Group GID: The attribute used to name an LDAP/AD group. For example, cn.
```
Change to 
```
LDAP Group GID: The attribute used to name an LDAP/AD group. For example, cn. This field cannot be empty.
```

### PR DESCRIPTION
**Pull request by a-mccarthy**
Signed-off-by: Abigail McCarthy <mabigail@vmware.com>

Fixes #181 

This should be updated in all release versions of the docs

### CODE REVIEW

File: docs/administration/configure-authentication/ldap-auth.md [hunk 1]
```diff
@@ -23,19 +23,19 @@ If you want to manage user authentication by using LDAP groups, you must enable
...
+   - **LDAP UID**: An attribute, for example `uid`, or `cn`, that is used to match a user with the username. If a match is found, the user's password is verified by a bind request to the LDAP/AD server.
    - **LDAP Scope**: The scope to search for LDAP/AD users. Select from **Subtree**, **Base**, and **OneLevel**.
-   
+
      ![Basic LDAP configuration](../../../img/ldap-auth.png)  
 1. If you want to manage user authentication with LDAP groups, configure the group settings.
-   - **LDAP Group Base DN**: The base DN from which to lookup a group in LDAP/AD. For example, `ou=groups,dc=example,dc=com`.
-   - **LDAP Group Filter**: The filter to search for LDAP/AD groups. For example, `objectclass=groupOfNames`. 
-   - **LDAP Group GID**: The attribute used to name an LDAP/AD group. For example, `cn`.  
+   - **LDAP Group Base DN**: The base DN from which to lookup a group in LDAP/AD. For example, `ou=groups,dc=example,dc=com`. This field cannot be empty.
```
Comment (stonezdj) [inline docs/administration/configure-authentication/ldap-auth.md:32]: This filed cannot be empty when ldap group feature is enabled.

File: docs/administration/configure-authentication/ldap-auth.md [hunk 2]
```diff
@@ -23,19 +23,19 @@ If you want to manage user authentication by using LDAP groups, you must enable
...
+   - **LDAP Group Filter**: The filter to search for LDAP/AD groups. for OpenLDAP: `objectclass=groupOfNames`. for Active Directory: `objectclass=group`. This field cannot be empty.
```
Comment (stonezdj) [inline docs/administration/configure-authentication/ldap-auth.md:33]: This field cannot be empty when LDAP group feature is enabled

File: docs/administration/configure-authentication/ldap-auth.md [hunk 3]
```diff
@@ -23,19 +23,19 @@ If you want to manage user authentication by using LDAP groups, you must enable
...
+   - **LDAP Group GID**: The attribute used to name an LDAP/AD group. For example, `cn`. This field cannot be empty.
```
Comment (stonezdj) [inline docs/administration/configure-authentication/ldap-auth.md:34]: This field cannot be empty when LDAP group feature is enabled.

**Comment by a-mccarthy** [comment]
@stonezdj thanks for reviewing, i updated the docs to include thee new wording

**Pull request merged**